tools/chocolateypkg/main.go

```diff
@@ -97,29 +99,29 @@ func replaceNuspec(dir, version string) error {
 	return nil
 }
 
-func generateSha256(f *os.File) (string, error) {
+func generateSha256(url string) (string, error) {
+	r, err := http.NewRequestWithContext(context.Background(), http.MethodGet, url, http.NoBody)
```

Comment:
this is a super important change here we were generating the checksum from the .msi currently generated at the disk, here I'm changing to download package and generate checksum from it.

Reply:
Nice! Was that the reason for the flakiness you spotted?